site/vue/tsconfig.app.json

```diff
@@ -5,6 +5,7 @@
   "compilerOptions": {
     "composite": true,
     "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.app.tsbuildinfo",
+    "types": ["jquery"],
```

Comment:
Is this needed for this PR?  If not, it would be better to do this in a separate PR which can be merged immediately.